### PR TITLE
Added twig examples for Video Document Editable

### PR DESCRIPTION
**Pull request by MyZik**

### CODE REVIEW

File: doc/Development_Documentation/03_Documents/01_Editables/38_Video.md [hunk 1]
```diff
@@ -95,6 +95,24 @@ In the configuration, you could also specify additional options for external ser
 </section>
 ```
 
+```twig
+<section id="campaign_video">
+    {{ pimcore_video('campaignVideo', {
+            width: 700,
+            height: 400,
+            youtube: [
```
Comment (dvesh3) [inline doc/Development_Documentation/03_Documents/01_Editables/38_Video.md]: please use curly braces for twig hashes

File: doc/Development_Documentation/03_Documents/01_Editables/38_Video.md [hunk 2]
```diff
@@ -95,6 +95,24 @@ In the configuration, you could also specify additional options for external ser
...
+                autoplay: true,
+                modestbranding: true
+            ],
+            vimeo: [
```
Comment (dvesh3) [inline doc/Development_Documentation/03_Documents/01_Editables/38_Video.md]: curly braces

File: doc/Development_Documentation/03_Documents/01_Editables/38_Video.md [hunk 3]
```diff
@@ -116,5 +134,25 @@ In the configuration, you could also specify additional options for external ser
...
+<html>
+<head>
+    <link href="http://vjs.zencdn.net/5.4.4/video-js.css" rel="stylesheet">
+</head>
+<body>
+    {{ pimcore_video('myVideo', {
+            thumbnail: 'example',
+            width: 400,
+            height: 300,
+            attributes: ['class': 'video-js custom-class', 'preload': 'auto', 'controls': '', 'data-custom-attr': 'my-test']
```
Comment (dvesh3) [inline doc/Development_Documentation/03_Documents/01_Editables/38_Video.md]: curly braces with line formatted properties

File: doc/Development_Documentation/03_Documents/01_Editables/38_Video.md [hunk 4]
```diff
@@ -95,6 +95,24 @@ In the configuration, you could also specify additional options for external ser
 </section>
 ```
 
+```twig
+<section id="campaign_video">
+    {{ pimcore_video('campaignVideo', {
+            width: 700,
+            height: 400,
+            youtube: {cd ..
```
Comment (dvesh3) [inline doc/Development_Documentation/03_Documents/01_Editables/38_Video.md]: cd..?

Comment (MyZik) [reply doc/Development_Documentation/03_Documents/01_Editables/38_Video.md]: yeap, terminal :D

**Comment by dvesh3** [comment]
can you please also add twig basic example here? thanks! https://github.com/pimcore/pimcore/blob/f1cd006b13d13a080a256503d082d1ebe7444942/doc/Development_Documentation/03_Documents/01_Editables/38_Video.md#basic-usage---a-local-asset

**Comment by dvesh3** [comment]
@MyZik thanks a lot 👍

**Pull request merged**